src/app/components/json.tsx

```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/react';
+import { bytesToHex } from '@stacks/common';
+import { Box } from '@stacks/ui';
+
+import { isTypedArray } from '@shared/utils';
+
+function parseJsonReadable(value: any) {
+  if (typeof value === 'bigint') return value.toString();
```

Comment:
could be a helper with the others, isString etc

Reply:
You had actually already made this. 🤣